src/chunk.spec.js

```diff
@@ -0,0 +1,15 @@
+import chunk from './chunk'
+
+// The chunk function should be tested to ensure all elements 
+// of the original array are returned in correctly sized chunks. 
+
+describe('Test suite for "chunk"-function', () => {
+    describe('Test simple array', () => {
+        // const arr = ['a', 'b']
+        it("Should return [['a', 'b'], ['c', 'd']], on input (['a','b', 'c','d'], 2)", () => {
+            console.log(chunk(['a', 'b', 'c', 'd'], 2));
+            // expect(chunk(['a', 'b', 'c', 'd'], 2)).toBe([['a', 'b'], ['c', 'd']])
```

Comment:
I would leave the failing test in, for me its ok if the testsuite fails in the end. Since that actually shows the true state of the libary. We could also fix the actual functions, but I don't think that is neccesary at this stage of the process.

Did you find a input with which the function deals correctly?